### DISCUSSION TITLE
Anyway to set environment variables and command arguments for Cargo?

### DISCUSSION THREAD

**Discussion by psionic12** [intellij-rust/intellij-rust#10882] [IntelliJ Rust Plugin Development]
Is the some place to set environment variables and command parameters for cargo, just like the way in Deployment->CMake?

**Comment by Trustpad-web** [comment]
Yes, in Visual Studio Code, you can set environment variables and command parameters for Cargo (the Rust package manager) in the launch.json file, similar to how you can set them for CMake in the "Deployment" section. You can add the "env" and "args" properties to the Cargo launch configuration to specify the environment variables and command-line arguments, respectively.
